guillotina_elasticsearch/__init__.py

```diff
@@ -6,9 +6,14 @@
 from guillotina.utils import get_dotted_name
 
 
+def default_refresh():
+    return False
+
+
 app_settings = {
     "elasticsearch": {
         "bulk_size": 50,
+        "refresh": "guillotina_elasticsearch.default_refresh",
```

Comment:
Every time we index a document we call this function so we can decide the value of 'refresh'. For example, as we discussed, in that function we can check the `request.query` for a parameter like `refresh=wait_for`.